### PR TITLE
Adds hijabs

### PR DESCRIPTION
**Pull request by redstryker**
Adds hijabs in every base color as a loadout item.

### CODE REVIEW

**Comment by OhEmGee** [comment]
Only if you add crucifixes and kipas as well :^)

Nah, I'm totally fine with this.

**Comment by Minijar** [comment]
Kind of an odd addition but whatever.

**Comment by OhEmGee** [comment]
>Kind of an odd addition but whatever.

it's the muslim agenda mini

**Comment by Zuhayr** [comment]
How is it odd to add a headscarf/veil? You do realize that there are a Hell of a lot of Muslim folks in the world, yes?

**Comment by redstryker** [comment]
@OhEmGee , I'm actually planning on adding Kippot and Turbans later on, so all in good time.

**Comment by Yoshax** [comment]
You could look into a way of having the base sprites as greyscale, and then add a way for the player to select a color that is applied, similar to the way hair is done.

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 1]
```diff
@@ -184,4 +184,88 @@
...
+	display_name = "Dark red hijab"
+	path = /obj/item/clothing/head/hijab/darkred
+
+/datum/gear/head/hijab/green
+	display_name = "Green hijab"
+	path = /obj/item/clothing/head/hijab/green
+
+/datum/gear/head/hijab/green
+	display_name = "Green hijab"
+	path = /obj/item/clothing/head/hijab/grey
```
Comment (PsiOmegaDelta) [inline code/modules/client/preference_setup/loadout/loadout_head.dm]: I believe the copy-paste demon been involved, as I have a feeling this grey hijab isn't intended to be under the green entry.

**Comment by AltAccount111** [comment]
@redstryker Will crucifixes be added?

**Comment by redstryker** [comment]
@AltAccount111 , Seeing that I am actually working on five different things right now, not in the near future, but perhaps.  Although, I'm hesitant to add crucifixes since there is no 'jewelry' system right now, and I think that making them basically ties would just be cheesy.  Plus, there are other 'Christian' items already in game.  Take the nun outfit, or all of the chaplain's clothes, for instance.  I'm just trying to add some diversity to the platform, and right now, adding crucifixes is the least of my worries.  I have plans to sprite/code other things before I do that, but dependent, I may get around to it.

**Pull request merged**